internal/storage/postsql/write.go

```diff
@@ -56,6 +56,21 @@ func (ws writeSession) InsertBinding(binding dbmodel.BindingDTO) dberr.Error {
 	return nil
 }
 
+func (ws writeSession) UpdateBinding(binding dbmodel.BindingDTO) dberr.Error {
```

Comment:
I wonder if we should add `updated_at` field to `bindings` table if we allow update operations.

Reply:
For now we have expiration_seconds plus expired_at that could be used to calculate that value. Lets' discuss it outside of this PR.